### PR TITLE
Cleanup debianization (Buster) [REVPI-1688]

### PR DESCRIPTION
**Pull request by iluminat23**

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
 Priority: optional
-Maintainer: Philipp Rosenberger <p.rosenberger@kunbus.com>
-Build-Depends: debhelper (>= 12)
-Standards-Version: 4.6.0
+Maintainer: KUNBUS GmbH <support@kunbus.com>
+Rules-Requires-Root: no
 Homepage: https://github.com/erminas/noderedrevpinodes-server
 Vcs-Browser: https://github.com/RevolutionPi/noderedrevpinodes-server
 Vcs-Git: https://github.com/RevolutionPi/noderedrevpinodes-server.git
+Build-Depends: debhelper-compat (= 12), dh-python, python3
```
Comment (naruxde) [inline debian/control:9]:
```suggestion
Build-Depends: debhelper-compat (= 12), dh-python
```
python3 is a dependency of dh-python. If the upstream project uses setup tools, we have to add python3-all, but they don't even have a setup.py file :D

**Review by naruxde — CHANGES_REQUESTED** [review]

**Review by RamiGspo — COMMENTED** [review]
Typo error in Commit [032be9b](https://github.com/RevolutionPi/noderedrevpinodes-server/pull/4/commits/032be9b24929b9aeb5b6403bd80ceb1ff514026a)  alwasy -> always

After that and what Sven commented, it looks good to me.

File: debian/control (hunk 1 above)
Comment (iluminat23) [reply debian/control:9]: Without the build-dep to python3 I get the following lintian error:
```
Now running lintian noderedrevpinodes-server_1.0.4-1+revpi10+1_armhf.changes ...
E: noderedrevpinodes-server source: missing-build-dependency-for-dh-addon python3 => python3:any | python3-all:any | python3-dev:any | python3-all-dev:any | dh-sequence-python3
```

**Review by RamiGspo — APPROVED** [review]

**Review by naruxde — DISMISSED** [review]

**Review by RamiGspo — DISMISSED** [review]

**Review by RamiGspo — DISMISSED** [review]

File: debian/control [hunk 2]
```diff
@@ -1,23 +1,24 @@
...
 Depends:
  ${misc:Depends},
- python3 (>= 3.5),
+ ${python3:Depends},
  python3-revpimodio2 (>= 2.5.1),
  python3-bcrypt (>= 3.1.2),
  python3-cryptography (>= 1.7.1),
  python3-websockets(>= 3.2-2),
- python3-distro(>=1.0.0)
+ python3-distro(>= 1.0.0)
```
Comment (naruxde) [inline debian/control]: Shouldn't we add a reformat commit, because we use at least two spaces as indention in our other packages?

**Review by naruxde — CHANGES_REQUESTED** [review]

File: debian/control (hunk 2 above)
Comment (RamiGspo) [reply debian/control]: I had to laugh... well played

**Review by RamiGspo — DISMISSED** [review]

File: debian/changelog [hunk 3]
```diff
@@ -1,3 +1,21 @@
...
+
+  * Use debhelper-compat instead of the compat file
+  * Avoid newer Standards-Version then provided by the debhelper version
+  * Reorder the control file contents to match our other packages
+  * Tell the package build we don't need root or fakeroot
+  * Use KUNBUS GmbH as maintainer
+  * Remove useless comments from rules file
+  * Drop --with-systemd from rules file
+  * Use dh_python for handling the installation
+  * Add missing space (non-functional change)
```
Comment (naruxde) [inline debian/changelog]:
```suggestion
  * Fix indention of Depends block
```

**Review by naruxde — CHANGES_REQUESTED** [review]

**Review by naruxde — APPROVED** [review]

**Pull request merged**